content/content-strategy/cms/cms-requirements/section2.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: text/textblock
+---
+Hosting and security is vital to any CMS, especially in government. This is an important part of choosing the right CMS. Things like where your website content is hosted and what levels of security are in place, should be considered as part of the choosing process.
+### [1]Service level agreement
```

Comment:
Incorrect value for display level. Should be `### [2]`

Reply:
Steve. Fixed.